cpp/harvest/remid.sh

```diff
@@ -1,47 +1,123 @@
 #!/bin/bash
-# This file's main purpose is documentation
+###########################################
+# Author: Steven Lolong (steven.lolong@uni-tuebingen.de)
+# copyright 2023 Tübingen University Library.  All rights reserved.
+# 
+# Script to generate a ixtheo_remid file
+###########################################
+
 set -o errexit -o nounset
 
+# related to creating temporary file
 declare -r FILE_ORI="remid.xml"
 declare -r FILE_TEMP="remid_tmp.xml"
-declare -r FILE_TEMP1="remid_tmp1.xml"
 declare -r FILE_TEMP_CLEAN="remid_tmp_clean.xml"
+declare -r GET_TOTAL_RECORDS="total_rec.xml"
+
+# the output file
 declare -r FILE_NEW="ixtheo_remid_`date +'%y%m%d'`_001.xml"
 
-# Download MARC XML from Hebis via SRU
+# related to file name pattern
+declare -r TEMP_FILE_NAME_PREFIX="tmp_file"
+declare -r CLEAN_FILE_NAME_PREFIX="clean_file"
+
+# maximum number of records per download 
+declare -r RECORDS_PER_CALL=100
```

Comment:
Please increase this to 500 and add the information to the comment above that 1000 is max